src/makeCalculator.js

```diff
@@ -4,7 +4,47 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+
+    // Метод для додавання
+    add: function (a) {
```

Comment:
[CODE KNOWLEDGE] - Instead of using the `function` keyword, use the method shorthand syntax for defining object methods. For example, change `add: function (a) {` to `add(a) {`.